celery/beat.py

```diff
@@ -237,7 +237,7 @@ def is_due(self, entry):
     def _when(self, entry, next_time_to_run, mktime=time.mktime):
         adjust = self.adjust
 
-        return (mktime(entry.schedule.now().timetuple()) +
+        return (mktime(entry._default_now().timetuple()) +
```

Comment:
This is a protected method. You should probably create instead a public method or getter in `ScheduleEntry`.

Reply:
Agreed. But a public method or getter would just duplicate (or call) ``_default_now``. So how about just making that public, i.e. merely removing the underscore and slightly refactoring?